django_freeradius/tests/test_api.py

```diff
@@ -52,3 +53,140 @@ def test_postauth_400(self):
         response = self.client.post(reverse('freeradius:postauth'), {})
         self.assertEqual(RadiusPostAuth.objects.all().count(), 0)
         self.assertEqual(response.status_code, 400)
+
+
+    def test_accounting_start(self):
+        RadiusAccounting.objects.all().count()
+-        self.assertEqual(RadiusAccounting.objects.all().count(), 0)
+        RadiusAccounting.objects.create(session_id='35000006', unique_id='75058e505ac30c5b4b9c686ee853b70f',
```

Comment:
@lillopaco I don't understand why you are creating the radacct object through the ORM, could you explain? I think this line should be removed